README.md

```diff
@@ -0,0 +1,54 @@
+# Portfolio-mobile-version
+Microverse project
+
```

Comment:
- [ ] OPTIONAL: I recommend adding a brief description about the project, remember that REAMDE is the first document that introduces your hard-work  to other users 💯